cliStocksTracker.py

```diff
@@ -9,13 +9,34 @@
 import contextlib
 import configparser
 import multiconfigparser
+import math
 
 import numpy as np
 import yfinance as market
 
 from matplotlib import colors
 from colorama import Fore, Style
 from datetime import datetime, timedelta
+from renderer import Renderer
+
+
+def merge_config(config, args):
+    if config["General"]["independent_graphs"]:
+        args.independent_graphs = config["General"]["independent_graphs"] == "True"
+
+    if config["Frame"]["width"]:
+        args.graph_width = int(config["Frame"]["width"])
+
+    if config["Frame"]["height"]:
+        args.graph_height = int(config["Frame"]["height"])
+
+    if config["General"]["timezone"]:
+        args.timezone = config["General"]["timezone"]
+
+    if config["General"]["rounding_mode"]:
+        args.rounding_mode = config["General"]["rounding_mode"]
+
+    return
```

Comment:
the config is nice but it requires all of the params to be stored as strings. I've created this merger, which admittedly, is a bit wonky but essentially the flow is this
- init all args with defaults
- if the config specifies a section, override the args with that
- only pass args around for source of truth for options